test/junit/src/test/scala/strawman/collection/LazyZipOpsTest.scala

```diff
@@ -64,7 +65,7 @@ class LazyZipOpsTest {
 
   @Test
   def lazyZip2_toIterable(): Unit = {
-    val iter: Iterable[(Int, Int)] = zipped2
+    val iter: collection.Iterable[(Int, Int)] = zipped2
```

Comment:
Why was this change needed?

Reply:
This is due to a difference in naming resolution. `strawman.collection.immutable._` is imported so here Dotty resolve `Iterable` to `strawman.collection.immutable.Iterable` but `zipped2` returns `strawman.collection.LazyZip2` which is implicitly converted to `View` which is a subtype of `strawman.collection.Iterable` and not `strawman.collection.immutable.Iterable`